wordpress-mobile/WordPress-iOS
# Add Media Editor as a pod

Part of #13251

This PR removes all the Media Editor references from WPiOS and add it through Cocoapods.

### To test

Just make sure that the Media Editor is opening.

1. In Aztec, tap an image
2. Tap Edit
3. Edit the image
4. Tap done

PR submission checklist:

- [x] I have considered adding unit tests where possible — the unit tests now lives in the MediaEditor repo. I'll configure CircleCI for it.
